src/index.js

```diff
@@ -1,15 +1,27 @@
-import _ from 'lodash';
 import './style.css';
 
+const displayList = document.getElementById('todolist');
 
-function component() {
-    const element = document.createElement('div');
-  
-    // Lodash, now imported by this script
-    element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-    element.classList.add('hello');
-  
-    return element;
-  }
-  
-  document.body.appendChild(component());
\ No newline at end of file
+const toDoList = [
+  {
+    description: 'wash the dishes',
+    completed: false,
+    index: null,
+  },
+  {
+    description: 'complete To Do list project',
+    completed: false,
+    index: null,
+  },
+];
+
+Object.keys(toDoList).forEach((item) => {
+  displayList.innerHTML += `
+  <div class="task ${toDoList[item].description}">
+  <input class="taskstatus" type="checkbox">
+  <p class="taskdescription">${toDoList[item].description}</p>
+  <span class="icon"></span>
+  </div>
+  <hr width="100%">
+  `;
+});
```

Comment:
- [ ] Please make sure to write a function that iterates over the todos array and populate the HTML, as per [this](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=Write%20a%20function%20to%20iterate%20over%20the%20tasks%20array%20and%20populate%20an%20HTML%20list%20item%20element%20for%20each%20task.) requirement
- [ ] Please make sure to sort the array of todos according to their index values before populating the DOM, as per [this](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=The%20list%20should%20appear%20in%20order%20of%20the%20index%20values%20for%20each%20task.) requirement.

Reply:
1. the function is on line 18, and its working fine. can you please elaborate on what should I fix?

Reply:
@maov19, What I meant was wrapping it inside of a function instead of having it like this. As this is a best practice to reduce repeating code, which was a requirement.

Reply:
Hi @ShadyShawkat, thanks for your feedback.
All the observations were solved. Please let me know if there is anything else I should work on. 🏁

Reply:
Got it, done!